cmd/awscollector/main.go

```diff
@@ -46,8 +47,22 @@ func main() {
 		log.Fatalf("failed to build components: %v", err)
 	}
 
+	var zapHooks []zap.Option
+
 	// init lumberFunc for zap logger
 	lumberHook := logger.GetLumberHook()
+	if lumberHook != nil {
+		zapHooks = append(zapHooks, zap.Hooks(lumberHook))
+	}
+
+	// init an ECS Error Logger and a go routine for error reporting when the STATUS_MESSAGE_FILE_PATH is set
+	if os.Getenv("STATUS_MESSAGE_FILE_PATH") != "" {
+		ecsErrorLogger := logger.NewECSErrorLogger()
+		ecsErrorHook := ecsErrorLogger.GetErrorHook()
+		zapHooks = append(zapHooks, zap.Hooks(ecsErrorHook))
+		ecsErrorLogger.Ticker = time.NewTicker(time.Minute)
```

Comment:
Should this interval be configurable?  Why is the main routine responsible for creating the ticker rather than the logger?  Not creating the ticker when the logger is created sounds like an incident waiting to happen.